src/main.rs

```diff
@@ -1,2 +1,42 @@
-fn main() {
+use warp::Filter;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+use geoengine_operators::Operator;
+
+
+#[tokio::main]
+async fn main() {
+    pretty_env_logger::init();
+
+    let workflow_registry = Arc::new(Mutex::new(HashMap::<usize, Operator>::new()));
+    let workflow_registry = warp::any().map(move || Arc::clone(&workflow_registry));
+
+    let register_workflow = warp::post()
+        .and(warp::path!("workflow" / "register"))
+        .and(warp::body::json())
+        .and(workflow_registry.clone())
+        .and_then(register_workflow);
+
+    let load_workflow = warp::get()
+        .and(warp::path!("workflow" / usize))
+        .and(workflow_registry.clone())
+        .and_then(load_workflow);
+
+    warp::serve(register_workflow.or(load_workflow)).run(([127, 0, 0, 1], 3030)).await
```

Comment:
Der Port müsste in Zukunft konfigurierbar sein.

Reply:
das stimmt ist aber ein anderer task